tests/evm/test_address.py

```diff
@@ -32,7 +32,7 @@ def test_address(address: U160):
         bytecode_table=set(bytecode.table_assignments()),
         rw_table=set(
             RWDictionary(9)
-            .call_context_read(1, CallContextFieldTag.CalleeAddress, address)
+            .call_context_read(1, CallContextFieldTag.CalleeAddress, Word(address))
```

Comment:
I think here we dont use U160 is to save cells. Because address op is just do context `read`. We just assure range check on `write` part, whearas in read part just need hi/lo 2 cell. instead of having U160 which will have 20 cells.

Reply:
This is the witness value for the rw table.  The RwTable only contains Words for Values, so I think it's good to make it explicit that we encode the address as a word.  The `Word` constructor takes the input as an integer, splits it into bytes, and puts the first 128 bits in lo and the rest in hi.